src/RuleSet/Sets/PERCS1x0RiskySet.php

```diff
@@ -42,10 +43,11 @@ public function getRules(): array
 
     public function getDescription(): string
     {
-        return <<<'DESC'
-            **This ruleset is deprecated** in favour of `@PER-CS2.0:risky`.
+        return 'Rules that follow `PER Coding Style 1.0 <https://www.php-fig.org/per/coding-style/>`_.';
+    }
 
-            Rules that follow `PER Coding Style 1.0 <https://www.php-fig.org/per/coding-style/>`_.
-            DESC;
+    public function getSuccessorsNames(): array
+    {
+        return ['@PER-CS2.0:risky'];
```

Comment:
I wonder if we can reference that ruleset without using ninja value. Basically to prevent typo , or mentioning the ruleset that was also already removed.

Best case: on code level.
Worst case: on tests

Reply:
Unfortunately `RuleSetDescriptionInterface::getName()` is not static, so we would need weird workarounds like `(new PERCS2x0Set())->getName()`. I can do it if you want, but currently we also reference other rulesets like this in child-sets, so not sure if this is a concern of this PR.

Reply:
At least a test to prevent usage of non-existing sets ?

Reply:
I believe it's out of the scope of this PR, let's create an issue for it and solve it separately.